lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -241,10 +241,12 @@ class ReportUIFeatures {
     scoreShadow.style.transform = `scaleY(${1 - scrollPct * 0.2})`;
 
     // Fade & move the scorescale
-    const scoreScalePositionDelta = 32;
-    const scoreScale = this._dom.find('.lh-scorescale', scoresContainer);
-    scoreScale.style.opacity = `${1 - scrollPct}`;
-    scoreScale.style.transform = `translateY(${scrollPct * -scoreScalePositionDelta}px)`;
+    try {
```

Comment:
I think we don't need this try/catch any longer.. pretty sure it was resolved somewhere else.. but IIRC it only applied in the single-category case.

Reply:
I think we still need it.  Even on master it throws errors when scrolling in single category reports.  I can split this into another PR if it doesn't belong here, but I think this is still a bug.
master:
![image](https://user-images.githubusercontent.com/6392995/50991021-88a46280-14c8-11e9-8735-c5ac52e35aa5.png)

Reply:
lets just move a few lines inside of a 

```js
const scoreScale = this._dom.find('.lh-scorescale', scoresContainer);
if (scoreScale) { ... }
```

Reply:
The error is on the find itself.  This still throws, no?

Reply:
what i meant to say is.. "what a beautiful try/catch! let's keep it just like it is."